charts/images.yaml

```diff
@@ -17,8 +17,8 @@ images:
     tag: v2.0.7
   - name: cilium-nodeinit
     sourceRepository: github.com/cilium/cilium
-    repository: gcr.io/google-containers/startup-script
-    tag: v1
+    repository: docker.io/cilium/startup-script
+    tag: af2a99046eca96c0138551393b21a5c044c7fe79
```

Comment:
We had some troubles with non semver tags, ref: https://github.com/gardener/gardener-extension-provider-alicloud/issues/127. I see that `v1`, `v2` and `v3` tags are available here https://hub.docker.com/r/cilium/startup-script/tags - can we use one of them ?

Reply:
v1, v2 and v3 are too old, they didn't publish a tagged release for the latest image (1.8.0) and they are using the image hash.

Reply:
tested with v3 and used it for now. Hopefully, they use tagged images in the future releases.

Reply:
@vpnachev : the image tags can be chosen arbitrarily. however, the dependency versions must be compliant to our "relaxed" semver